FirebaseSessions/ProtoSupport/Protos/sessions.proto

```diff
@@ -0,0 +1,160 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+// Instructions for updating this file:
+//
+// 1. Get the session_event.proto from the backend
+// 2. Copy everything below the options, and past it here below the package
+//    declaraction.
+// 3. Remove all the datapol.semantic_type tags
+// 4. Remove "wireless_android_play_playlog." from
+//    "wireless_android_play_playlog.NetworkConnectionInfo"
+// 5. Any new proto fields of type string, repeated, or bytes must be specified
+//    in the sessions.options file as "type:FT_POINTER"
+//
+
+syntax = "proto3";
+
+import "clientanalytics.proto";
```

Comment:
This can be avoided by including the definitions here.

Reply:
See my comment on proto2 vs 3